include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMeanShort.hpp

```diff
@@ -0,0 +1,91 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Kepler_BrouwerLyddaneMeanShort__
+#define __OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Kepler_BrouwerLyddaneMeanShort__
+
+#include <OpenSpaceToolkit/Core/Containers/Pair.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/Shared.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Coordinate/Frame.hpp>
+#include <OpenSpaceToolkit/Physics/Coordinate/Position.hpp>
+#include <OpenSpaceToolkit/Physics/Coordinate/Velocity.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Duration.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived/Angle.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Length.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMean.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/COE.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+namespace orbit
+{
+namespace models
+{
+namespace kepler
+{
+
+using ostk::core::ctnr::Pair;
+using ostk::core::types::Real;
+using ostk::core::types::Shared;
+using ostk::core::types::String;
+
+using ostk::math::obj::Vector6d;
+
+using ostk::physics::coord::Frame;
+using ostk::physics::coord::Position;
+using ostk::physics::coord::Velocity;
+using ostk::physics::time::Duration;
+using ostk::physics::units::Angle;
+using ostk::physics::units::Derived;
+using ostk::physics::units::Length;
+
+using ostk::astro::trajectory::orbit::models::kepler::COE;
+using ostk::astro::trajectory::orbit::models::kepler::BrouwerLyddaneMean;
+
+/// @brief                      Brouwer-Lyddane Mean Short Orbital Elements. Short periodic variations are averaged.
+///
+/// @ref
+/// https://space.stackexchange.com/questions/22151/whats-a-brouwer-lyddane-mean-semi-major-axis-or-any-other-for-an-orbit-in-a-l
+
+class BrouwerLyddaneMeanShort : public BrouwerLyddaneMean
+{
+   public:
+    BrouwerLyddaneMeanShort(
+        const Length &aSemiMajorAxis,
+        const Real &anEccentricity,
+        const Angle &anInclination,
+        const Angle &aRaan,
+        const Angle &anAop,
+        const Angle &aMeanAnomaly
+    );
+
+    virtual COE toCOE() const override;
+
+    static BrouwerLyddaneMeanShort Cartesian(
+        const COE::CartesianState &aCartesianState, const Derived &aGravitationalParameter
+    );
+
+    static BrouwerLyddaneMeanShort Undefined();
```

Comment:
Worth moving at `BrouwerLyddaneMean` level?

Reply:
As it's a static constructor, it will return a parent type, so unless we want to return pointers I believe this is simplest.

Reply:
Ah yup, I see. Thanks!